src/components/vm/snapshots/vmSnapshotsCard.jsx

```diff
@@ -99,14 +99,14 @@ export class VmSnapshotsCard extends React.Component {
             {
                 name: _("Creation time"), value: (snap, snapId) => {
                     const date = prettyTime(snap.creationTime);
-                    return (<div className="snap-creation-time">
-                        <div id={`${id}-snapshot-${snapId}-date`}>
+                    return (<Flex className="snap-creation-time">
+                        <FlexItem id={`${id}-snapshot-${snapId}-date`} spacer={{ default: 'spacerSm' }}>
                             {date}
-                        </div>
-                        { snap.isCurrent && <Tooltip content={_("Current")}>
-                            <i id={`${id}-snapshot-${snapId}-current`} className="pficon pficon-ok" />
-                        </Tooltip> }
-                    </div>);
+                        </FlexItem>
+                        { snap.isCurrent && <FlexItem><Tooltip content={_("Current")}>
+                            <CheckIcon id={`${id}-snapshot-${snapId}-current`} color="#6EC664" />
```

Comment:
How did you find this color? I see here the success color is #3E8635 (alert icon background) https://www.patternfly.org/v4/guidelines/colors ?

Reply:
Why not just doing `<CheckIcon color="green" />`?

Reply:
That would work!

Reply:
It's "green-300" on that page. "green" is too dark, I tried "lightgreen" and "pf-color-green-300" but that didn't change anything. I'd be much happier about a symbolic name, of course.

Reply:
The only other way to change the color is to pass a class with className and that class can set:
color: pf-color-green-300

I really don't like color code in the JSX but I am happy to send a followup for this.